js/src/common/components/LoadingIndicator.tsx

```diff
@@ -40,21 +42,30 @@ export interface LoadingIndicatorAttrs extends ComponentAttrs {
  *
  * - `containerClassName` Class name(s) to apply to the indicator's parent
  * - `className` Class name(s) to apply to the indicator itself
- * - `size` Size of the loading indicator
+ * - `size` Size of the loading indicator (small, medium or large)
  * - `containerAttrs` Optional attrs to be applied to the container DOM element
+ * - `block` Displays the spinner as a block element with a height of 100px
+ * - `inline` Displays the spinner as an inline element, centrally aligned vertically.
  *
  * All other attrs will be assigned as attributes on the DOM element.
  */
 export default class LoadingIndicator extends Component<LoadingIndicatorAttrs> {
   view() {
-    const { size, ...attrs } = this.attrs;
+    const { inline = false, block = false, size = 'medium', containerClassName, className, ...attrs } = this.attrs;
```

Comment:
Why not default to either block or inline? Also, shouldn't these be mutually exclusive?

Reply:
I didn't want to default to one or the other as there may be some cases where extensions wish to do their own custom styling.

Maybe this would be better as a `display` attribute on the component, such as `display="block"` and `display="inline"`?